common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -534,6 +534,8 @@
 
                 .speed-option,
                 .control-lang {
+                  @include border-left($baseline/10 solid rgb(14, 166, 236));
```

Comment:
@clrux FYI, I'm just looking at the video code as i'm not expert in a11y or styling.